dwave_networkx/drawing/chimera_layout.py

```diff
@@ -190,30 +190,30 @@ def _xy_coords(i, j, u, k):
 
 
 def draw_chimera(G, **kwargs):
-    """Draws graph G in a Chimera cross topology.
+    """Draw graph ``G`` in a Chimera cross topology.
 
-    If `linear_biases` and/or `quadratic_biases` are provided, these
-    are visualized on the plot.
+    Linear and quadratic biases are visualized on the plot as specified
+    in the ``linear_biases`` and ``quadratic_biases`` parameters.
 
     Parameters
     ----------
     G : NetworkX graph
-        Should be a Chimera graph or a subgraph of a Chimera graph.
+        :term:`Chimera` :term:`graph` or a :term:`subgraph` of a Chimera graph.
 
     linear_biases : dict (optional, default {})
-        A dict of biases associated with each node in G. Should be of
-        form {node: bias, ...}. Each bias should be numeric.
+        A dict of biases associated with each node in ``G`` and of
+        the form ``{node: bias, ...}``. Each bias is numeric.
 
     quadratic_biases : dict (optional, default {})
-        A dict of biases associated with each edge in G. Should be of
-        form {edge: bias, ...}. Each bias should be numeric. Self-loop
+        A dict of biases associated with each edge in ``G`` and of
+        the form ``{edge: bias, ...}``. Each bias is numeric. Self-loop
         edges (i.e., :math:`i=j`) are treated as linear biases.
 
     kwargs : optional keywords
-       See networkx.draw_networkx() for a description of optional keywords,
-       with the exception of the `pos` parameter which is not used by this
-       function. If `linear_biases` or `quadratic_biases` are provided,
-       any provided `node_color` or `edge_color` arguments are ignored.
+       Parameters in :func:`~networkx.drawing.nx_pylab.draw_networkx`, except for the ``pos`` parameter.
```

Comment:
Does it work with the simpler path `networkx.draw_networkx`, it will be less fragile to changes in structure than `networkx.drawing.nx_pylab.draw_networkx`.

Reply:
Also don't think so, ``networkx.drawing.nx_pylab.draw_networkx  reference/generated/networkx.drawing.nx_pylab.draw_networkx.html#networkx.drawing.nx_pylab.draw_networkx`` is the only ``draw_networkx`` under ``py:function`` in the NetworkX ``objects.inv`` file that I see

Reply:
Removed per @arcondello.